### ISSUE TITLE
ionic serve --lab fails with error Cannot GET //__ion-dev-server/ionic_lab.html

### ISSUE THREAD

**Issue by faraazc** [driftyco/ionic-cli#1666]
#### Short description of the problem:
ionic serve --lab fails with error Cannot GET //__ion-dev-server/ionic_lab.html
However ionic serve works (i.e. without lab option)
This is happening after upgrading ionic cli to 2.1.8 and ionic framework to 2.0.0-rc.2 and App scripts to 0.0.43

http://localhost:8100//__ion-dev-server/ionic_lab.html Failed to load resource: the server responded with a status of 404 (Not Found)

#### What behavior are you expecting?
i am expecting ios, android and windows previews

**Steps to reproduce:**
1. upgrade app to use ionic cli to 2.1.8 and ionic framework to 2.0.0-rc.2 and App scripts to 0.0.43
2. run command sudo ionic serve --lab on terminal
3. see the output on  browser window

```
insert any relevant code between the above and below backticks
```

**Post the output of `ionic info` below please**
Ionic Framework Version: 2.0.0-rc.2
Ionic CLI Version: 2.1.8
Ionic App Lib Version: 2.1.4
Ionic App Scripts Version: 0.0.43
ios-deploy version: Not installed
ios-sim version: Not installed
OS: macOS Sierra
Node Version: v6.7.0

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

**Comment by savanvadalia** [comment]
Exactly same issue when running with ``` ionic serve --lab``` on windows

Here is my system info

Cordova CLI: 6.4.0
Ionic Framework Version: 2.0.0-rc.2
Ionic CLI Version: 2.1.8
Ionic App Lib Version: 2.1.4
Ionic App Scripts Version: 0.0.43
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Windows 10
Node Version: v6.9.1
Xcode version: Not installed

**Comment by savanvadalia** [comment]
Changing the scripts in ```package.json```  as below works fine. Please note the difference of the word **ionic**.
Don't know the implications of doing this. I found the solution from here
https://forum.ionicframework.com/t/ionic-serve-lab-dont-work-in-rc2/69862/8


FROM
```
  "scripts": {
    "ionic:build": "ionic-app-scripts build",
    "ionic:serve": "ionic-app-scripts serve"
  },
```
TO
 ``` 
"scripts": {
    "build": "ionic-app-scripts build",
    "serve": "ionic-app-scripts serve"
}  
```

**Comment by FrancescoMussi** [comment]
This solution worked for me:

```
"scripts": {
  "build": "ionic-app-scripts build",
  "watch": "ionic-app-scripts watch",
  "serve:before": "watch",
  "emulate:before": "watch",
  "deploy:before": "build",
  "build:before": "build",
  "run:before": "watch"
},
```

SO link:

https://stackoverflow.com/questions/40561794/cannot-get-ion-dev-server-ionic-lab-html-in-ionic-2-1-8

**Comment by jthoms1** [comment]
This is resolved in relase 0.0.44 of @ionic/app-scripts. Please update and also ensure you are using the latest version of the CLI.

`npm install --save-dev @ionic/app-scripts@latest`
`npm install -g ionic@latest`

**Comment by omar300** [comment]
sudo npm install --save-dev  --unsafe-perm node-sass
